client/web/src/search/suggestion/SmartSearchListItem.tsx

```diff
@@ -0,0 +1,58 @@
+import { mdiArrowRight } from '@mdi/js'
+import classNames from 'classnames'
+
+import { SyntaxHighlightedSearchQuery } from '@sourcegraph/branded'
+import { formatSearchParameters } from '@sourcegraph/common'
+import { SearchPatternType } from '@sourcegraph/shared/src/graphql-operations'
+import { ProposedQuery } from '@sourcegraph/shared/src/search/stream'
+import { Link, createLinkUrl, Icon, Text } from '@sourcegraph/wildcard'
+
+import styles from './QuerySuggestion.module.scss'
+
+interface SmartSearchListItemProps {
+    proposedQuery: ProposedQuery
+    previewStyle?: boolean
+}
+
+const processDescription = (description: string): string => {
+    const split = description.split(' ⚬ ')
+
+    split[0] = split[0][0].toUpperCase() + split[0].slice(1)
+    return split.join(', ')
+}
+
+export const SmartSearchListItem: React.FunctionComponent<SmartSearchListItemProps> = ({
+    proposedQuery,
+    previewStyle = false,
+}) => (
+    <li key={proposedQuery.query} className={classNames(previewStyle ? 'py-2' : styles.listItem)}>
+        <Link
+            to={createLinkUrl({
+                pathname: '/search',
+                search: formatSearchParameters(new URLSearchParams({ q: proposedQuery.query })),
+            })}
+            className={classNames(previewStyle ? 'text-decoration-none' : styles.link)}
+        >
+            <Text className="mb-0">
+                <span
+                    className={classNames(previewStyle ? 'text-muted' : styles.listItemDescription)}
+                >{`${processDescription(proposedQuery.description || '')}`}</span>
```

Comment:
I think can unwrap this template string
```suggestion
                >{processDescription(proposedQuery.description || '')}</span>
```